tests/test_TFNetworkLayer.py

```diff
@@ -1867,7 +1865,8 @@ def test_SplitDimsLayer_batch_feature_major_keep_feature():
       "output": {'class': 'split_dims', 'from': 'data', 'axis': 'T', 'dims': [-1, 1]}  # [B,D,T,1]
     })
     out = net.get_default_output_layer().output
-    assert_equal(out.get_dim_tag(2), net.extern_data.get_default_input_data().get_time_dim_tag())
+    assert out.get_dim_tag(2).is_equal(
+      net.extern_data.get_default_input_data().get_time_dim_tag(), derived_matches=True)
```

Comment:
`derived_matches=True` should not be needed here. The splitting with `[-1, 1]` should yield the same dim tag (under standard equality).

Reply:
Hm okay. Then I'll have to change some things around in the logic of `SplitDimsLayer`.